app/server-utils/stampy.ts

```diff
@@ -156,6 +174,31 @@ export const loadAllQuestions = withCache('allQuestions', async () => {
   return rows.map(({name}) => name)
 })
 
+const extractMainQuestion = (question: string | Entity | null): string | null => {
+  switch (typeof question) {
+    case 'string':
+      return question.replace(/```/g, '')
+    case 'object':
+      return question?.name || null
+    default:
+      return question
+  }
+}
+const toTag = (r: CodaRow): Tag => ({
+  rowId: r.id,
+  tagId: r.values['Tag ID'],
+  name: r.name,
+  url: r.browserLink,
+  internal: r.values['Internal?'],
+  questions: r.values['Questions tagged with this'].map(({name}) => name),
```

Comment:
this is just a list of questions titles, as the query doesn't return the UI IDs - one of the following will need to be done at some point:
* expand this query to also get the UI IDs here
* mix this with `loadAllQuestions()` to make sure the ids are known
* update the tags with the relative ids when an actual question is viewed
* worry about it when someone clicks on a tag

The first 2 solutions will save overall queries/latency at the cost of more stuff needed to be stored on the server. Probably won't be a problem until there are a few million questions... They also make the first loading of the site a bit longer (subsequent ones are fine, as the results are cached)
The other options are nicer in that they're explicit with how they utilise magic, which is always a bonus